Global/GlobalDefines.h

```diff
@@ -66,10 +66,10 @@ CLANG_DIAG_ON(deprecated)
 //#define foreach Q_FOREACH
 
 
-typedef boost::uint32_t U32;
```

Comment:
unrelated change. file a different PR if you think this is useful

Reply:
The later versions of Shiboken were ignoring the `boost::*_t` types. Given that C++11 has fixed width integers it compiled without nuisance. Can remain here in the case of being accepted, otherwise I can file a different PR.